packages/react-utils/package.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "@onaio/react-utils",
```

Comment:
Does this need to include `react` as a dependency / peerDependency?

Reply:
Yes I think we should have it as a peer dependency